tools/tfrecord2idx

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env python
 import sys
 import struct
+import tensorflow as tf
+import tensorflow_io as tfio
```

Comment:
I'm afraid it is not used.

Reply:
This is needed. Otherwise we will see ``File system scheme 's3' not implemented``. Because s3 interface is defined in tensorflow/io `s3_filesystem.h`.

Reply:
But doesn't `tf.io` just import this under the hood? I think this the C/C++ header where it is defined should not matter and not translate to python imports?

Reply:
No, after we open the file using ``tf.io.gfile.GFile``, we will see error for ``f.read()`` and any other File IO. In tensorflow package, only GCS and POSIX FS support are added. S3 support was moved from tensorflow to tensorflow_io starting from ``version==2.6.0``. If we only import tensorflow, we cannot find the S3 definition in C++ shared library, because that's defined in tensorflow_io package.

If you run the script on your end without importing tensorflow_io, you will see failures at ``f.read()``.

See the following two comments from tensorflow members:
https://github.com/tensorflow/tensorflow/issues/51583#issuecomment-902786636
https://github.com/tensorflow/tensorflow/issues/51583#issuecomment-902332632

Reply:
Thank you for detailed exaplantaiton. In this case I think we need to
```suggestion
import tensorflow_io as tfio  # noqa:F401
```
To silence flake8 warnings as it won't detect that this import is important indeed.

Reply:
Applied the changes, thank you!

Reply:
Thanks!